app/views/home/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+<% @photos_low_res.find_each do |photo| %>
+  <div class="col-xs-6 col-md-3">
```

Comment:
I didn't know that, but I discovered that `col-xs-6` has been dropped in bootstrap 4 in favor of `col-6`:

http://v4-alpha.getbootstrap.com/layout/grid/#grid-options